### PR TITLE
upgrade pytorchvideo to 0.1.2

### PR DESCRIPTION
**Pull request by aniketmaurya**
## What does this PR do?

Fixes # (issue)

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by aniketmaurya** [comment]
hi @tchaton, pls review the PR for `pytorchvideo` version upgrade.

File: flash/video/classification/data.py [hunk 1]
```diff
@@ -44,12 +44,13 @@
 if _PYTORCHVIDEO_AVAILABLE:
     from pytorchvideo.data.clip_sampling import ClipSampler, make_clip_sampler
     from pytorchvideo.data.encoded_video import EncodedVideo
-    from pytorchvideo.data.encoded_video_dataset import EncodedVideoDataset, labeled_encoded_video_dataset
+    # from pytorchvideo.data.encoded_video_dataset import EncodedVideoDataset, labeled_encoded_video_dataset
```
Comment (Borda) [inline flash/video/classification/data.py:47]:
```suggestion
```

**Review by Borda — APPROVED** [review]

File: flash/video/classification/data.py (hunk 1 above)
Comment (aniketmaurya) [reply flash/video/classification/data.py:47]: Oops! I forgot to remove that line 😅

**Review by ethanwharris — COMMENTED** [review]
Hi @aniketmaurya this looks good, although you have a test failure, mind taking a look?

**Comment by aniketmaurya** [comment]
Sure @ethanwharris, I'll check it today evening

**Comment by aniketmaurya** [comment]
Hi @ethanwharris, the test was failing due to `annotation` argument is required in `ClipSampler`. Not sure if this was there in previous version.
I looked at the [source code of `pytorchvideo`](https://github.com/facebookresearch/pytorchvideo/blob/master/pytorchvideo/data/clip_sampling.py#L27) but the `annotation` arg is not being used by any of the implemented Sampler 😀

**Review by ethanwharris — APPROVED** [review]
@aniketmaurya Haha, thanks for sorting it out :smiley: LGTM!

**Pull request merged**